libs/langchain/langchain/agents/agent.py

```diff
@@ -194,6 +196,7 @@ def tool_run_logging_kwargs(self) -> Dict:
         return {}
 
 
+@deprecated("0.1.0", removal="0.2.0")
```

Comment:
for this, can we say to use openai_tools_agent?

Reply:
thought about that, only issue is tools requires latest models. but maybe that's fine?

Reply:
oh this is the base agent — seems like openai tools is too specific for this?

was referring to openai multi functions agent being slightly different from openai tools agent in comment above

Reply:
think for abstract base classes probably don't need alternative?